src/App.vue

```diff
@@ -50,21 +50,22 @@
       attach="#app"
     />
 
-    <!-- Initial Page Load Transition -->
+    <!-- Loading Dashboard spinner -->
     <v-fade-transition>
-      <div class="loading-container" v-show="showLoadingContainer">
+      <div class="loading-container" v-show="showLoadingDashboardSpinner">
         <div class="loading__content">
           <v-progress-circular color="primary" size="50" indeterminate />
           <div class="loading-msg" v-if="!isSignoutRoute">Loading Dashboard</div>
         </div>
       </div>
     </v-fade-transition>
 
-    <!-- Alternate Loading Spinner -->
+    <!-- Fetching Data spinner -->
     <v-fade-transition>
-      <div class="loading-container grayed-out" v-show="showSpinner">
+      <div class="loading-container grayed-out" v-show="showFetchingDataSpinner">
         <div class="loading__content">
           <v-progress-circular color="primary" size="50" indeterminate />
+          <div class="loading-msg white--text">Fetching Data</div>
```

Comment:
This spinner was also on the FilingHistoryList, so I changed the flag to a store variable and just show it here.